### PR TITLE
JupyterLab 3 update

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

**Comment by martinRenou** [comment]
This appears to be ready. There is a compilation issue that could be an issue with `@jupyterlab/codemirror` but I am really not sure.

File: README.rst [hunk 1]
```diff
@@ -42,13 +42,6 @@ RobotKernel can be installed using the usual Python package manager tools, like
 
    $ pip install robotkernel
 
-For JupyterLab it is recommended to also install the Robot Framework syntax highlighting and Jupyter widgets support:
-
-.. code:: bash
-
-   $ jupyter labextension install jupyterlab_robotmode
-   $ jupyter labextension install @jupyter-widgets/jupyterlab-manager
```
Comment (martinRenou) [inline README.rst:50]: This will not be needed anymore once JupyterLab 3 and ipywidgets supports it

**Comment by martinRenou** [comment]
This PR is ready, it can be merged before the JupyterLab 3 release if we make a major release.

**Comment by martinRenou** [comment]
@corker would it be fine to merge and release this? This would need a major release I suppose

**Review by corker — APPROVED** [review]

**Pull request merged**